submit-web/src/components/SubmissionItem/ConsultationRecord/ConsultationRecordStaffView/NotesSection.tsx

```diff
@@ -0,0 +1,142 @@
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Box,
+  Button,
+  TextField,
+  Typography,
+} from "@mui/material";
+import KeyboardArrowRightIcon from "@mui/icons-material/KeyboardArrowRight";
+import { BCDesignTokens } from "epic.theme";
+import EventNoteIcon from "@mui/icons-material/EventNote";
+import Note from "./Note";
+import { When } from "react-if";
+import { useState } from "react";
+
+export default function NotesSection() {
+  const mockNotes = [
```

Comment:
We should remove these right?

Reply:
sure yea

Reply:
im going to leave it as a empty array since were mapping over it for the notes section